tests/test_vector2_reflect.py

```diff
@@ -38,7 +38,7 @@ def test_reflect_prop(initial: Vector2, normal: Vector2):
     assert initial.isclose(returned)
     note(f"initial ⋅ normal: {initial * normal}")
     note(f"reflected ⋅ normal: {reflected * normal}")
-    assert isclose((initial * normal), -(reflected * normal))
+    assert isclose((initial.dot(normal)), -(reflected.dot(normal)))
```

Comment:
Why is this change needed?

Reply:
Because I changed it before discovering `@override` and didn't feel like switching it back.

Reply:
Yet the code right above it still uses `*`.
Please switch back to `*` for consistency (and also because it would be hella weird not to use the preferred notation in tests)